README.md

```diff
@@ -266,10 +266,20 @@ Full API reference documentation can be found in the
 
 ## Communication
 
-The Sceptre community uses a Slack channel #sceptre on the og-aws Slack for
-discussion. To join use this link <http://slackhatesthe.cloud/> to create an
-account and join the #sceptre channel.
+Sceptre community discussions happen in the #sceptre chanel in the
+[og-aws Slack](https://github.com/open-guides/og-aws).  To join click
+on <http://slackhatesthe.cloud/> to create an account and join the
+#sceptre channel.
 
 ## Contributing
 
 See our [Contributing Guide](CONTRIBUTING.md)
+
+
+## Sponsors
+
+[![GoDaddy](sponsors/godaddy_logo.png "GoDaddy")](https://www.godaddy.com)
+
+[![Cloudreach](sponsors/cloudreach_logo.png "Cloudreach")](https://www.cloudreach.com)
```

Comment:
Do they still sponsor anything? I thought they dropped all involvement...

Reply:
Yes, they have however the docker for sceptre is deployed to their dockerhub account.  Once it's move to the sceptreorg account then i believe we can drop cloudwatch.   More info in issue #1098

Reply:
I would recommend at least keeping a historical note about Cloudreach's involvement/sponsorship in the long term.

Reply:
done